framework/include/binary_manager/binary_manager.h

```diff
@@ -38,16 +38,14 @@
  ****************************************************************************/
 #ifdef CONFIG_BINMGR_UPDATE
 /**
- * @brief Loads the binary after downloading a new binary
+ * @brief Loads the binaries after downloading new binaries
  * @details @b #include <binary_manager/binary_manager.h>\n
- *  This function loads a new binary after downloading the binary.\n
- * It requests the binary manager to load the input binary name.
- * @param[in] binary_name The name of a new binary to be loaded
+ *  This function loads a new binaries after downloading the binaries.\n
```

Comment:
Could you please change this to "This function loads the new binaries after downloading them"